tracks/csharp/exercises/nucleotide-count/mentoring.md

```diff
@@ -0,0 +1,57 @@
+### Reasonable solutions
+
+### Using a foreach loop
+
+```csharp
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+public static class NucleotideCount
+{
+    public static IDictionary<char, int> Count(string sequence)
+    {
+        var result = new Dictionary<char, int> { {'A', 0}, {'C', 0}, {'G', 0}, {'T', 0} };
+        
+        foreach (var c in sequence)
+        {
+            if (result.ContainsKey(c)) result[c]++;
```

Comment:
```suggestion
            if (result.ContainsKey(c))
                result[c]++;
```

Having the statement of the if-clause on the same line is not something that one sees used often in C#.  Maybe put it on the next line (like [bob](https://github.com/exercism/website-copy/blob/master/tracks/csharp/exercises/bob/mentoring.md))?

Reply:
Happy to take it off the same line, but if so I'll use brackets because I find bracketless to be misleading and possibly dangerous (Some discussion of that here: https://softwareengineering.stackexchange.com/questions/145896/the-bracketless-if-statement-is-frowned-upon-but-is-it-still-bad-style-if-your)

Reply:
A good example of problems caused, in part, by bracketless IFs is Apple's 2014 SSL vulnerability: https://blog.codecentric.de/en/2014/02/curly-braces/

Reply:
Sure, feel free to add brackets. That is most idiomatic.

Reply:
Resolved in 61a32fd4aefa4877c053be4d36cc1b7bffa696b8